en/whats-new-in-v1.1.md

```diff
@@ -0,0 +1,49 @@
+---
+title: What's New in v1.1
+summary: Learn the new features in TiDB Operator v1.1.
+category: introduction
+---
+
+# What's New in v1.1
+
+Based on v1.0, TiDB Operator v1.1 has several new features, including TiDB 4.0 support, TiKV data encryption, and TLS certificate configuration, etc. TiDB Operator v1.1 also supports deploying new components such as TiFlash and TiCDC.
```

Comment:
Latin abbreviation is not recommended in technical documentation. `Including` here means inexhaustive and the list can actually go on.
```suggestion
Based on v1.0, TiDB Operator v1.1 has several new features, including TiDB 4.0 support, TiKV data encryption, and TLS certificate configuration. TiDB Operator v1.1 also supports deploying new components such as TiFlash and TiCDC.
```